sw/airborne/firmwares/rotorcraft/guidance/guidance_indi_hybrid.c

```diff
@@ -229,13 +249,24 @@ void guidance_indi_run(float *heading_sp) {
   float pos_x_err = POS_FLOAT_OF_BFP(guidance_h.ref.pos.x) - stateGetPositionNed_f()->x;
   float pos_y_err = POS_FLOAT_OF_BFP(guidance_h.ref.pos.y) - stateGetPositionNed_f()->y;
   float pos_z_err = POS_FLOAT_OF_BFP(guidance_v_z_ref - stateGetPositionNed_i()->z);
-
+  
+  // float scheduled_pos_gain = gih_params.pos_gain + approaching_rope * 0.8;
+  // float scheduled_pos_gainz = gih_params.pos_gainz + approaching_rope * 0.8;
+  scheduled_pos_gain = gih_params.pos_gain + approaching_rope * GUIDANCE_INDI_DELTA_KP;
```

Comment:
Would be more convenient if this was just a selection

Reply:
done